### DISCUSSION TITLE
How does fre:ac compare to 'exact audio copy'?

### DISCUSSION THREAD

**Discussion by Source-Force** [enzo1982/freac#319] [Help & Support]
I would like to digitize my large CD collection. The quality of the digitized CDs is especially important to me. I did some research and people kept recommending a program called "exact audio copy" (EAC), but it only runs on Windows. 
However, I neither use Windows as a matter of principle nor Wine, so I was looking for a program that runs on GNU/Linux and found fre:ac. IMHO it is the best choice for GNU/Linux so far.  

From what I understand, EAC is praised for its use of AccurateRip. I found this option in fre:ac (Verifiers) as well. Furthermore, there is a "Full cdparanoia mode" which seems to repair scratches. 

After activating these two opitions, I wonder: what does EAC has that fre:ac does not? 
Is EAC really producing higher quality rips, i.e. less errors, in ripped tracks than fre:ac? 
What is the reason people still widely recommend EAC over fre:ac? 
If AccurateRip is used in fre:ac, why is it not listed as an AccurateRip compatible program on the AccurateRip page (http://www.accuraterip.com/software.htm)? 

Thank you in advance!

**Comment by enzo1982** [comment]
Thank you for choosing fre:ac!

Yes, fre:ac supports AccurateRip. And when your rip is successfully verified with AccurateRip (you will find a _"Track has been accurately ripped"_ message in the log in that case), the rip is just as good as one from EAC. It's accurately ripped and you got exactly the audio samples that are supposed to be on the CD. Cannot do any better.

EAC, however, has some advantages when AccurateRip verification fails. First, it checks not only the data at the currently configured drive offset against the AccurateRip database, but in case that fails can also try if the ripped data verifies using a different offset. That way, different pressings of a CD can be verified by it which would currently still fail to verify with fre:ac. I plan to add this feature to fre:ac at some point, though.

Second, EAC can rip a CD multiple times and automatically compare the results. So when AccurateRip verification cannot be performed, e.g. because the disc is not in the database, you can at least be sure that your drive gives consistent results. With fre:ac you would have to do that manually.

So when you see the _"Track has been accurately ripped"_ message in fre:ac's log, you are good. But when AccurateRip verification is not possible, EAC still has some advantages.

fre:ac not being listed on the AccurateRip page is just because I never asked for it to be included there. There are many more AccurateRip-capable tools also not listed there.

By the way, I would enable cdparanoia mode only as a last resort. When sectors cannot be read because of scratches, it repairs them by interpolating over the unavailable data. That means you will not here a clicking noise where the scratch was, but the audio will still be distorted. Also, cdparanoia's jitter correction can in some rare cases be wrongly triggered and then damage an otherwise fine rip. For these reasons (and because it's really slow), cdparanoia mode is disabled by default in fre:ac.

**Comment by zordaz** [comment]
I think both EAC and Fre;ac have their value. Main advantages from my experiences:
- Fre;ac is natively available for Linux, EAC only runs via Wine. 
- Fre:ac is often faster than EAC.
- EAC tends to be a bit better on my system when having problematic discs. 
- EAC has CTDB support available.
- EAC logfiles are IMO a bit friendlier to read.

There is no reason to drop one for the other. In my workflow I use Fre:ac first for ripping CD's and only try EAC or CUETools when having problems with a disc.
